src/current/cockroachcloud/serverless-cluster-management.md

```diff
@@ -49,11 +49,11 @@ The monthly cost estimate is calculated using simple extrapolation that assumes
 1. In the **Usage this month** section of your cluster's [**Overview** page](cluster-overview-page.html), click **Estimate usage cost**.
 1. Select a time frame in which your workload was active.
 
-    Your used [RUs](learn-about-request-units.html), used storage, and spend from the selected time frame will be shown along with your monthly cost estimate. The spend and cost estimate do not account for the [free resources](learn-about-pricing.html#free-vs-paid-usage) granted to every organization, which you would have to use up before being charged.
+    Your used [RUs](plan-your-cluster-serverless.html#request-units), used storage, and spend from the selected time frame will be shown along with your monthly cost estimate. The spend and cost estimate do not account for the [free resources](plan-your-cluster-serverless.html#free-vs-paid-usage) granted to every organization, which you would have to use up before being charged.
```

Comment:
Your monthly estimate might show 20M RUs, but you won't actually be charged because 50M RUs can be deducted from your invoice each month. So you can mentally subtract the free resources from your cost estimate, but you shouldn't actually set your resource limit any lower. If you set a 10M RU limit, that's all you can use, even though more RUs could have been used for free. Will think about this...

Reply:
Is it more clear to say something like "this is the cost estimate before your monthly deduction is applied"?